go.sum

```diff
@@ -0,0 +1,79 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
This whole file == same as the go.mod comment.

Reply:
I'll get rid of this file as well.